emukit/examples/vanilla_bayesian_quadrature_with_rbf/vanilla_bq_loop_with_rbf.py

```diff
@@ -12,7 +12,8 @@
 
 
 def create_vanilla_bq_loop_with_rbf_kernel(X: np.ndarray, Y: np.ndarray, integral_bounds: List,
-                                           rbf_lengthscale: np.float=1.0, rbf_variance: np.float=1.0):
+                                           rbf_lengthscale: float=1.0, rbf_variance: np.float=1.0) -> \
```

Comment:
huh?

Reply:
I have to guess what you mean because the comment is not very descriptive, but I believe you are referring to the float. It seems I changed the first one but not the second one. Will fix.